testing/proxy/proxy_test.go

```diff
@@ -0,0 +1,213 @@
+// package proxy is an example of how the testing package can be used. It tests
+// three contracts that are used to form a proxy pattern. An explanation of
+// proxy contracts in Solidity can be found here:
+// https://www.cyfrin.io/blog/upgradeable-proxy-smart-contract-pattern
+package proxy
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"testing"
+
+	"github.com/kwilteam/kwil-db/common"
+	"github.com/kwilteam/kwil-db/core/utils"
+	kwilTesting "github.com/kwilteam/kwil-db/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// Test_Impl_1 tests the impl_1.kf file.
+func Test_Impl_1(t *testing.T) {
+	kwilTesting.RunSchemaTest(t, kwilTesting.SchemaTest{
+		Name:        "impl_1",
+		SchemaFiles: []string{"./impl_1.kf"},
+		SeedStatements: map[string][]string{
+			"impl_1": {
+				`INSERT INTO users (id, name, address)
+				 VALUES ('42f856df-b212-4bdc-a396-f8fb6eae6901'::uuid, 'satoshi', '0xAddress'),
+				 ('d68e737d-708f-45f8-9311-317afcaccc63'::uuid, 'zeus', 'zeus.eth')`,
+			},
+		},
+		TestCases: []kwilTesting.TestCase{
+			{
+				// should create a user - happy case
+				Name:     "create user - success",
+				Database: "impl_1",
+				Target:   "create_user",
```

Comment:
wow, if we can execute this in REPL would be very handy.

Reply:
what do you mean in REPL?

Reply:
directly interact with engine

Reply:
Ah I see. Yeah that would be pretty easy to do.